{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=62.0", "setuptools-git-versioning<2"]
+req uires = ["setuptools>=62.0", "setuptools-git-versioning<2"]
```

Comment:
this is a bug, but it will likely be fixed on the other PR, so don't change it here or it will raise a conflict.